app/assets/stylesheets/authors.css.scss

```diff
@@ -0,0 +1,13 @@
+.author {
+  clear: both;
+  .author_books {
+    list-style: none;
+    .book {
+      float: left;
+      padding-left: 20px;
+      img {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4